### DISCUSSION TITLE
auro-library: introduce script to automate library wide changes

### DISCUSSION THREAD

**Discussion by Izook** [AlaskaAirlines/WC-Generator#504] [Announcements]
### Is your feature request related to a problem? Please describe.

It would be lovely if there was a script I could use to make an identical change across all of our component repos. 

### Describe the solution you'd like

Provide a customizable bash script that could make changes across all of our component repos. 

### Describe alternatives you've considered

Manually making changes across all of our repos.

**Comment by blackfalcon** [comment]
There are ways to do this, but it depends on the change. And this is not typically recommended. There was a bash I wrote that would update all the repos issues labels for example. At a certain scale, it get cumbersome and buggy. 

The discussions we have been having is around building solutions that allow for updates to be added to a repo upon the next build (e.g. markdown magic) or building the update in a deliverable form like a npm package that can be versioned and easily updated. 

I like the idea of making auro-library a consumable npm that makes it really easy to update internal scripts that are housed in this package. 

To that end, we need to properly segment things that are templates and scripts away from things that are direct consumables so that the npm package is not heavy with unnecessary files.
